CRM/Banking/PluginImpl/Importer/XML.php

```diff
@@ -315,7 +327,12 @@ protected function filterMatches($payment_node, $filter) {
    */
   protected function import_payment($payment_spec, $payment_node, $stmt_data, $index, $params) {
     $config = $this->_plugin_config;
-    $progress = ((float)$index / (float) $params['total_tx_count']);
+    $total_tx_count = (float) $params['total_tx_count'] ?? 1.0;
+    if ($total_tx_count) {
```

Comment:
Is this condition necessary now that there's a null-coalescing default of `1.0` and an implicit default of `0` due to the type cast to `float`?

Reply:
This might also use some brackets, too. If I get that right, he type cast will always cause a non-`NULL` value be given to the null-coalescing operator, making it rather useless here.